cmd/start.go

```diff
@@ -46,6 +47,32 @@ This command will start a stack and run it in the background.
 			}
 		}
 
+		dockerCmd := exec.Command("docker", "-v")
+		stdout, err := dockerCmd.Output()
+
+		if err != nil {
+			fmt.Println(err.Error())
+			return err
+		}
+		fmt.Println(string(stdout))
```

Comment:
I don't think we need to print the output from the command here (and elsewhere this PR does this)

Reply:
Done. Removed the print statements, only returning the error now